frontend/src/features/contacts/contact/create/components/address/Address.tsx

```diff
@@ -128,20 +158,29 @@ interface IProvinceOrCountryName {
   provinces: SelectOption[];
   formLabels: Dictionary<string>;
   namespace?: string;
+  disabled?: boolean;
 }
 
 const ProvinceOrCountryName: React.FunctionComponent<IProvinceOrCountryName> = ({
   selectedCountry,
   provinces,
   formLabels,
   namespace,
+  disabled = false,
 }) => {
   if (selectedCountry === CountryCodes.Other) {
-    return <Input field={withNameSpace(namespace, 'countryOther')} label="Country name" />;
+    return (
```

Comment:
confused why this is being added in this PR, was this missed in the original implementation?